tangobot_app/app/src/main/java/com/ekumen/tangobot/application/MainActivity.java

```diff
@@ -576,4 +606,34 @@ public Transform getDeviceTransform() {
             return DefaultRobotTfPublisherNode.TRANSFORM_IDENTITY;
         }
     }
+
+    @Override
+    public void onSaveMapServiceCallFinish(boolean b, String s) {
+
+    }
+
+    @Override
+    public void onTangoConnectServiceFinish(int i, String s) {
+
+    }
+
+    @Override
+    public void onTangoDisconnectServiceFinish(int i, String s) {
+
+    }
+
+    @Override
+    public void onTangoReconnectServiceFinish(int i, String s) {
+
+    }
+
+    @Override
+    public void onGetMapUuidsFinish(List<String> list, List<String> list1) {
+
+    }
+
+    @Override
+    public void onTangoStatus(int i) {
+
+    }
```

Comment:
Since this is still in flux, and we are not using any of the disconnect and reconnect service calls yet, can we skip this suggestion?